packages/fuels-core/src/lib.rs

```diff
@@ -247,10 +245,10 @@ impl Tokenizable for bool {
 }
 
 impl Tokenizable for String {
-    fn from_token(token: Token) -> Result<Self, InstantiationError> {
+    fn from_token(token: Token) -> Result<Self, Error> {
         match token {
             Token::String(data) => Ok(data),
-            other => Err(InstantiationError(format!(
+            other => Err(Error::InstantiationError(format!(
```

Comment:
In the Bool tokenization this was changed to return `InvalidData` while here it returns `InstantiationError` for the same cause.


I'm not sure which one is better, but we should definitely choose one.